packages/core/src/Stores/gtm-store.js

```diff
@@ -13,7 +13,12 @@ import { getMT5AccountType }  from './Helpers/client';
 import { getAppId }           from '../config';
 
 export default class GTMStore extends BaseStore {
-    is_gtm_applicable = /^(16303|16929)$/.test(getAppId());
+    //The below line is commented to for testing purposes and should be uncommented before pushing to stageing/production
+    
+    // is_gtm_applicable = /^(16303|16929|19111|19112)$/.test(getAppId());
+
+    //the below line should be removed before pushing to staging/production 
+    is_gtm_applicable = true;
```

Comment:
This should be uncommented as per the comment.

Reply:
done